app/views/govuk_publishing_components/components/_step_by_step_nav_header.html.erb

```diff
@@ -10,28 +10,25 @@
   ]
   breadcrumb_presenter = GovukPublishingComponents::Presenters::Breadcrumbs.new(breadcrumbs)
 
-  data = {}
-  data[:module] = "ga4-link-tracker" unless disable_ga4
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
+  component_helper.add_class("gem-c-step-nav-header")
 
   unless disable_ga4
+    component_helper.add_data_attribute({ module: "ga4-link-tracker" })
     ga4_data = {
       event_name: "navigation",
       type: "super breadcrumb",
       index_link: "1",
       index_total: "1",
     }.to_json
   end
-
-  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
```

Comment:
This looks fine but I was a bit confused by it being in two separate commits. Maybe worth either merging them or adding something to the commit message to clarify that the shared helper is no longer needed as the margin bottom is now provided by the component wrapper helper?

Reply:
Good point - will do, thanks!